doc-enhancement/update PATH variable (#7367)
microsoft/vscode-docs: docs/cpp/config-mingw.md

1. Enter `Y` when prompted whether to proceed with the installation.

1. Add the path of your MinGW-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
   1. In the Windows search bar, type **Settings** to open your Windows Settings.
   1. Search for **Edit environment variables for your account**.
   1. In your **User variables**, select the `Path` variable and then select **Edit**.
   1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you used the default settings above, then this will be the path: `C:\msys64\ucrt64\bin`.

>    1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.

Reviewer: ```suggestion
   1. Select **OK**, and then select **OK** again in the **Environment Variables** window to update the `PATH` environment variable.
```

>    1. You will have to open a NEW console window for the updated PATH to work.

Reviewer: ```suggestion
      You have to reopen any console windows for the updated `PATH` environment variable to be available.
```
